x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityEsqlIT.java

```diff
@@ -284,7 +287,7 @@ public void wipeData() throws Exception {
         wipe.accept(client());
     }
 
-    @AwaitsFix(bugUrl = "cross-clusters query doesn't work with RCS 2.0")
+    @SuppressWarnings("unchecked")
     public void testCrossClusterQuery() throws Exception {
```

Comment:
nit: it would also be nice to include a test with two remote clusters

Reply:
done in https://github.com/elastic/elasticsearch/pull/107079/commits/02e1c25856e8bd9996f58128fbc4921711a82eb0

Reply:
I don't think that is necessary, there shouldn't be anything unique with ES|QL and the cardinality of clusters since this all flows through the remote client (and isn't really a security concern)